common/djangoapps/util/json_request.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, resp_obj=None, status=None, encoder=EDXJSONEncoder,
         elif isinstance(resp_obj, QuerySet):
             content = serialize('json', resp_obj)
         else:
-            content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=False)
+            content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=True)
```

Comment:
Well, it is not bizarre. It is solving a lot of mysterious bugs in CMS/LMS related to i18n.
- ORA updates impossible with LANG!=en_en
- files names with accents present in module store but export or printing problems in LMS 
....

I found this comment with git blame because I wanted to make sure why this was my fix


I noticed that some of the work was done in async with JSON requests in the CMS. My hypothesis is content are translated using gettext returning unicode strings, while xblock coders codes may have treated strings as strings natively (which they must be naïvely). The ensure asci = True (default value) use a \u escaping and normalizing and always gives keys as str. While (I guess) otherwhise the native type was kept and that a huge lot of encode/decode was made.

I know LTS has this fix, this is just a message to the next coder who will be puzzled to understand why this obscure patch solves l10n problems